extensions/MozillaIAM/Extension.pm

```diff
@@ -143,6 +145,30 @@ sub oauth2_client_handle_redirect {
   return;
 }
 
+sub object_end_of_set_all {
```

Comment:
Are there other kinds of ways that the `User` object can be modified that we should guard against?